### DISCUSSION TITLE
Decoding RadioHead-ASK ASCII

### DISCUSSION THREAD

**Discussion by cskinner74** [merbanan/rtl_433#2321] [Q&A] [answered]
Hi there! I'm working on a project where I'm trying to pass barometric data from a sensor on an Arduino to WeeWX via rtl_433.

Currently, when using the RadioHead-ASK library in Arduino, it's passing the data as a payload of ASCII characters, as seen here:
![image](https://user-images.githubusercontent.com/5474519/212709183-c5a8d34d-72e7-4a8c-aa11-2dcceb4abae7.png)

I'm not sure if there's a way to structure this data where it could be used by WeeWX, either in the way I'm sending from Arduino, or in the rtl_433 software, but I'd like to be able to get the data into a field:value setup similar to the way the supported weather stations appear. Example:
![image](https://user-images.githubusercontent.com/5474519/212709528-a36fc0e6-04aa-4b8f-a972-3798514bb229.png)

Is this something that is possible? Or does this even make sense?

**Comment by merbanan** [comment, marked as answer]
Hi, the RadioHead-ASK protocol does not contain this key. You need to transmit a protocol that implements this if you want decoding to work out of the box. Otherwise you need to send the payload to a consumer that can decode your values back to whatever format you want.

**Reply by cskinner74** [reply]
That makes sense, thanks for the quick response!
